app/_src/gateway/production/deployment-topologies/hybrid-mode/incremental-config-sync.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Incremental Configuration Sync
+---
+
+In hybrid mode, whenever you make changes to {{site.base_gateway}} entity configuration on the Control Plane, it immediately triggers a cluster-wide update of all Data Plane configurations. 
+In these updates, {{site.base_gateway}} sends the entire configuration set to the Data Planes - therefore, the bigger your configuration set is, the more time it takes to send and process, and the more memory is consumed proportional to the configuration size.
+
+You can enable **incremental configuration sync** to address this issue. 
+When entity configuration changes, instead of sending the entire configuration set for each change, {{site.base_gateway}} only sends the parts of the configuration that have changed. 
+
+<!--vale off-->
+{% mermaid %}
+flowchart TD
+
+A[Client]
+B(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Control Plane)
+C(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Data Plane)
+D(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Data Plane)
+E[Client]
+F(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Control Plane)
+G(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Data Plane)
+H(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Kong Data Plane)
+
+ subgraph id1 [<b>With</b> incremental config sync]
+ direction TB
+
+ E --"POST Route config
+ 1 entity
+ A few KB"---> F --"Route config
+ 1 entity
+ A few KB"---> G & H
+ end
+
+ subgraph id2 [<b>Without</b> incremental config sync]
+ direction TB
+
+ A --"POST Route config
+ 1 entity
+ A few KB"---> B --Route config
+ 30k entities
+ 30MB---> C & D
+
+ end
+
+style id1 stroke-dasharray:3,rx:10,ry:10
+style id2 stroke-dasharray:3,rx:10,ry:10
+linkStyle 0,1,2,3 stroke:#2e8e05,color:#2e8e05
+linkStyle 4,5 stroke:#d44324,color:#d44324
+
+{% endmermaid %}
+<!--vale on-->
+> _**Figure 1**: In an environment with 30k entities of about 30MB total, sending a POST request to update one entity sends the whole 30MB config to every Data Plane. 
+With incremental config sync enabled, that same POST request only triggers an update of a few KB._
+
+Incremental config sync achieves significant memory savings and CPU savings. 
+This means lower total cost of ownership for {{site.base_gateway}} users, shorter config propagation delay, and less impact to proxy latency. 
+See our [blog on incremental config sync](https://konghq.com/blog/product-releases/incremental-config-sync-tech-preview) for the performance comparisons.
```

Comment:
Is there a reason why we can't pull the table into this doc or the performance doc? I'd prefer the table lives here unless it fluctuates a lot, which is why we want to link to the blog instead.

Reply:
That's the answer ^ that table applies to the tech preview 3.8 image at a specific time. It won't be exactly the same for 3.10, and it will change with every release.